lib/test.js

```diff
@@ -1,18 +1,20 @@
 const http = require('http');
 const clc = require('chalk');
+var childProcess = require('child_process');
 
 // Date string for logging.
 function ds() {return (new Date()).toISOString() + " [test]   ";};
 
-function commands(commandarr,catalog,exit_if_fail) {
+async function commands(commandarr,catalog,exit_if_fail) {
 	for (var i = 0;i < commandarr.length;i++) {
 		command = commandarr[i].command;
 		if (exit_if_fail) {
 			console.log(ds() + "Running " + catalog + " test command " + i + ".");
 		}
 		try {
-			var out = require('child_process')
-				.execSync(command,{'stdio':['pipe','pipe','pipe']});		
+			
```

Comment:
Replaced execSync with exec and made it asynchronous. Now all the subsets tests are passing.

Reply:
Add link referencing bug this works around.